src/Http/routes.php

```diff
@@ -1,7 +1,16 @@
 <?php
 
-Route::group(['namespace' => 'Ourgarage\Contacts\Http\Controllers'], function () {
+Route::group(['middleware' => 'web', 'namespace' => 'Ourgarage\Contacts\Http\Controllers'], function () {
 
-    Route::get('/contacts', 'ContactsController@index');
+    Route::get('/contacts', 'ContactsController@index')->name('contacts::index');
+
+    Route::get('/admin/contacts', 'ContactsController@adminContactsIndex')->name('contacts::admin::contactsIndex')->middleware('auth');
+    Route::get('/admin/contacts/create', 'ContactsController@adminContactsCreate')->name('contacts::admin::contactCreate')->middleware('auth');
```

Comment:
Instead of assigning `auth` middleware to every route, you can put it to group inside this group. You can nest groups, as far as I know.